application/ticket-app-api/src/main/java/com/uket/app/ticket/api/service/TicketingService.java

```diff
@@ -91,4 +95,14 @@ private TicketStatus checkTicketStatus(Users user, University university) {
         }
         return TicketStatus.BEFORE_PAYMENT;
     }
+
+    public List<CheckTicketDto> checkUserTickets(Long userId) {
+        List<Ticket> tickets = ticketService.findAllTicketsByUserId(userId);
+        List<CheckTicketDto> list = new ArrayList<>();
+        for (Ticket ticket : tickets) {
+            CheckTicketDto from = CheckTicketDto.from(ticket);
+            list.add(from);
+        }
+        return list;
+    }
```

Comment:
유저의 모든 티켓을 가져오는 메서드인 것 같은데 `TicketingService`과는 관련없어 보여요..! 다른 클래스로 옮기는게 좋을 것 같습니당

그리고 `checkUserTickets`라는 메서드명이 잘 이해가 안되는 것 같아요. 단순히 티켓들을 조회하는 기능만 수행하고 Ticket을 체크하는 기능은 없는 것 같은데 혹시 이렇게 명명하신 이유가 있을까요?

```suggestion
    public List<CheckTicketDto> checkUserTickets(Long userId) {
        List<Ticket> tickets = ticketService.findAllTicketsByUserId(userId);
        
        return tickets.stream().map(CheckTicketDto::from).toList();
    }
```
이런 식으로 Stream을 사용해서 코드를 줄일 수도 있으니까 참고 부탁드려요~

Reply:
userTicket들의 정보를 반환한다 생각해서 저렇게 지었습니다. 다른 이름 고려해보도록 하겠습니다.
또한 클래스 따로 만들어서 옮겨두도록 하겠습니다!